conjur/api/api.py

```diff
@@ -113,7 +113,7 @@ def authenticate(self) -> str:
         """
         if not self.login_id or not self.api_key:
             # TODO: Use custom error
```

Comment:
you can remove this TODO since we now have a custom error :)

Reply:
removed